sample/006-multi-db-typeorm-app/src/health/health.controller.ts

```diff
@@ -0,0 +1,20 @@
+import { Controller, Get } from '@nestjs/common';
+import { HealthCheck, HealthCheckService, TypeOrmHealthIndicator } from '@nestjs/terminus';
+import { getConnection } from 'typeorm';
+
+@Controller('health')
+export class HealthController {
+  constructor(
+    private health: HealthCheckService,
+    private db: TypeOrmHealthIndicator,
+  ) {}
+
+  @Get()
+  @HealthCheck()
+  check() {
+    return this.health.check([
+      async () => this.db.pingCheck('db1Connection', { connection: getConnection('db1Connection')}),
+      async () => this.db.pingCheck('db2Connection', {connection: getConnection('db2Connection')}),
```

Comment:
Wouldn't it make more sense to use `@InjectConnection('db2Connection')` in the constructor instead? 

https://github.com/nestjs/typeorm/blob/master/lib/common/typeorm.decorators.ts#L16

Reply:
@Tony133 I think you've missed this message :)

Reply:
example updated with @InjectConnection --> 9a4dd2c